charts/external-dns/templates/validation.tpl

```diff
@@ -0,0 +1,6 @@
+{{/*
+Mutually exclusive txtPrefix and txtSuffix
+*/}}
+{{- if and .Values.txtPrefix .Values.txtSuffix -}}
+  {{- fail (printf "'txtPrefix' and 'txtSuffix' mutually exclusive") -}}
```

Comment:
This would be a new pattern in the chart and I think if we're going to fail this needs to be part of the template to make it as easily understandable as possible. See above.

Reply:
Removed validation.tpl